UICatalog/Scenarios/VkeyPacketSimulator.cs

```diff
@@ -81,31 +83,32 @@ public override void Setup ()
 			Win.Add (outputVerticalRuler);
 
 			var tvOutput = new TextView {
+				Title = "Output",
 				X = 1,
 				Y = Pos.Bottom (outputHorizontalRuler),
 				Width = Dim.Fill (),
 				Height = Dim.Fill (),
 				ReadOnly = true
 			};
 
+			// Detect unknown keys and reject them.
 			tvOutput.KeyDown += (s, e) => {
-				//System.Diagnostics.Debug.WriteLine ($"Output - KeyDown: {e.KeyEvent.Key}");
-				e.Handled = true;
-				if (e.KeyEvent.Key == Key.Unknown) {
+				//System.Diagnostics.Debug.WriteLine ($"Output - KeyDown: {e.Key}");
+				//e.Handled = true;
+				if (e.KeyCode == KeyCode.Unknown) {
 					_wasUnknown = true;
 				}
 			};
 
-			tvOutput.KeyPressed += (s, e) => {
+			tvOutput.KeyUp += (s, e) => {
 				//System.Diagnostics.Debug.WriteLine ($"Output - KeyPress - _keyboardStrokes: {_keyboardStrokes.Count}");
 				if (_outputStarted && _keyboardStrokes.Count > 0) {
-					var ev = ShortcutHelper.GetModifiersKey (e.KeyEvent);
-					//System.Diagnostics.Debug.WriteLine ($"Output - KeyPress: {ev}");
-					if (!tvOutput.ProcessKey (e.KeyEvent)) {
-						Application.Invoke (() => {
-							MessageBox.Query ("Keys", $"'{ShortcutHelper.GetShortcutTag (ev)}' pressed!", "Ok");
-						});
-					}
+					//// TODO: Tig: I don't understand what this is trying to do
```

Comment:
If the key wasn't handled by the `TextView` will popup a Dialog with the keys pressed. e.g. `Ctrl+Shift+Alt+B` but now all the keys pressed together aren't work now and I get this exception.

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/658592d5-6e46-44e4-bfa1-ea8e9fdbd677)

Reply:
Now on `WindowsDriver` we can't output `é` because we can't cancel on the `KeyDown` the `´` where before the `KeyPress` was only invoked when the `e` is pressed. Now all the keys are immediately processed without waiting for the next key. When we press `´` the terminal wait for the next key and if the next key pressed is a non supported to combining the key, like the `w` will output `´w`. I know this is complex but is how it work.
The `NetDriver` and the `CursesDriver` are output twice.